Assets/Scripts/Grid/Positioning/SpiralSequenceRandomPositionProvider.cs

```diff
@@ -0,0 +1,109 @@
+using System.Collections.Generic;
+using Math.Random;
+using UnityEngine;
+
+namespace Grid.Positioning
+{
+    /// <summary>
+    /// Implementation of <see cref="IRandomGridPositionProvider"/> which returns a set with no duplicates.
+    /// It returns a list of random unique positions walking through the grid from the center outwards in a
+    /// spiral pattern, deciding at every step to pick that position or not by a probability range (30%) 
+    /// </summary>
+    public class SpiralSequenceRandomPositionProvider : IRandomGridPositionProvider{
```

Comment:
add a space before the curly brace:

`IRandomGridPositionProvider {`

Reply:
Almost